storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/query/Query.java

```diff
@@ -103,9 +102,7 @@ public String toString() {
     /**
      * Builder for {@link Query}
      */
-    public static class Builder {
-
-        Uri uri;
+    public static final class Builder {
```

Comment:
Why did you decide to make it final? Can it be useful for user to extend it for some custom behavior. For example receive `String` instead of `Uri`?